documentation/docs/20-core-concepts/20-load.md

```diff
@@ -322,8 +322,8 @@ export async function load({ fetch, setHeaders }) {
 	const url = `https://cms.example.com/products.json`;
 	const response = await fetch(url);
 
-	// cache the page for the same length of time
-	// as the underlying data
+	// During SSR, this universal load function runs on the server.
+	// The headers are set on the HTML response, caching the page.
```

Comment:
```suggestion
	// The headers are set on the HTML response, caching the page
	// for the same length of time as the underlying data
```

Reply:
I wanted to emphasize this part in the code example:

> When running in the browser, `setHeaders` has no effect.

Could this be emphasized? This was the part I was struggling with.

---

If universal load server runs on the server
→ it is SSR (not page invalidation, etc.)
→ the response is HTML
→ headers are set on the HTML response

Reply:
I would rewrite it to say "Headers are only set during SSR, caching the page ..."

Reply:
Love it! Just changed the 'page' to 'HTML'.

I know SSR means server-side rendering, which results in a HTML response.

But I wanted to clarify that this is a HTML response, not a 'page' load function response in a JSON format.